app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/AttachmentOptions.kt

```diff
@@ -57,65 +53,49 @@ import com.wire.android.util.permission.rememberOpenFileBrowserFlow
 import com.wire.android.util.permission.rememberOpenGalleryFlow
 import com.wire.android.util.permission.rememberRecordAudioRequestFlow
 import com.wire.android.util.permission.rememberTakePictureFlow
+import com.wire.android.util.ui.KeyboardHeight
 
 @Composable
-fun AttachmentOptions(
+fun AttachmentOptionsComponent(
     onAttachmentPicked: (UriAsset) -> Unit,
+    onRecordAudioMessageClicked : () -> Unit,
     tempWritableImageUri: Uri?,
     tempWritableVideoUri: Uri?,
     isFileSharingEnabled: Boolean,
     modifier: Modifier = Modifier
 ) {
-    Box(
-        modifier
-    ) {
-        Divider(color = MaterialTheme.wireColorScheme.outline)
-        AttachmentOptionsComponent(
-            onAttachmentPicked,
+    Box(Modifier.height(KeyboardHeight.DEFAULT_KEYBOARD_TOP_SCREEN_OFFSET)) {
```

Comment:
The `modifier` parameter is unused now.

Shouldn't we allow the callers of this Composable to customise the modifier?